components/image-builder-mk3/pkg/orchestrator/orchestrator.go

```diff
@@ -556,7 +556,7 @@ func (o *Orchestrator) getAbsoluteImageRef(ctx context.Context, ref string, allo
 		return "", status.Errorf(codes.InvalidArgument, "cannt resolve base image ref: %v", err)
 	}
 
-	ref, err = o.RefResolver.Resolve(ctx, ref, resolve.WithAuthentication(auth))
+	ref, err = o.RefResolver.Resolve(ctx, ref, resolve.WithAuthentication(auth), resolve.WithSkipTLSVerify(o.Config.SkipTLSVerify))
```

Comment:
This should not be necessary because we mount the custom CA cert in the image-builder container. Digging into why this is still required.